src/capitalsFirst.js

```diff
@@ -17,8 +17,29 @@
  *
  * @returns {string} - string with uppercase words in front
  */
+
+function isUpperCase(firstLetter) {
```

Comment:
isUpperCase - not need to know which letter you pass(first/last or something else), just `letter`